certora/specs/ERC20.spec

```diff
@@ -0,0 +1,256 @@
+// This is spec is taken from the Open Zeppelin repositories at https://github.com/OpenZeppelin/openzeppelin-contracts/blob/448efeea6640bbbc09373f03fbc9c88e280147ba/certora/specs/ERC20.spec, and patched to support the DelegationToken.
+// Note: the scope of this specification encompases only the implementation contract and not the interactions with the implementation contract and the proxy.
+
+definition nonpayable(env e) returns bool = e.msg.value == 0;
+definition nonzerosender(env e) returns bool = e.msg.sender != 0;
+
+methods {
+    function name()                                external returns (string)  envfree;
+    function symbol()                              external returns (string)  envfree;
+    function decimals()                            external returns (uint8)   envfree;
+    function totalSupply()                         external returns (uint256) envfree;
+    function balanceOf(address)                    external returns (uint256) envfree;
+    function allowance(address,address)            external returns (uint256) envfree;
+    function owner()                               external returns address   envfree;
+    function delegatedVotingPower(address)         external returns uint256   envfree;
+    function delegatee(address)                    external returns address   envfree;
```

Comment:
```suggestion
    function owner()                               external returns (address)   envfree;
    function delegatedVotingPower(address)         external returns (uint256)   envfree;
    function delegatee(address)                    external returns (address)   envfree;
```

Reply:
I think @colin-morpho wanted to stay as close to the original file as possible, to make it easy to highlight the diff

Reply:
yes, I missed this. We need to settle on the discussion on slack before moving forward with this PR then